src/validation.cpp

```diff
@@ -1851,6 +1851,18 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_NULLDUMMY;
     }
 
+    // SEGWIT2X signalling.
+    if ( VersionBitsState(pindex->pprev, chainparams.GetConsensus(), Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE &&
```

Comment:
> I've asked this before, but is the orphaning really necessary? Once 80% support Segwit2x, isn't it likely 15 of the remaining 20% support BIP141?

If you did not start orphaning the holdouts at 80%, a 5% minority not part to the agreement could still effectively veto BIP141, so it is necessary to ensure the lowered activation threshold.